pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -117,7 +196,7 @@ func (l *podNICImpl) setPodInterfaceCache(iface *v1.Interface, podInterfaceName
 	return nil
 }
 
-func (l *podNICImpl) readIPAddressesFromLink(podInterfaceName string) (string, string, error) {
+func (l *podNIC) readIPAddressesFromLink(podInterfaceName string) (string, string, error) {
```

Comment:
This `podInterfaceName` is an member of the struct. You can drop it from the signature, can't you ?

Reply:
Done